src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -11,63 +13,33 @@
     @Test
     public void excercise1() {
         // 1. Open test site by URL
-        String expectedURL = "https://jdi-testing.github.io/jdi-light/index.html";
-        webDriver.navigate().to(expectedURL);
         String actualURL = webDriver.getCurrentUrl();
-        softAssert.assertEquals(actualURL, expectedURL);
+        softAssert.assertEquals(actualURL, TestProperties.getTestProperties().getProperty("homePageURL"));
 
         // 2. Assert Browser title
         softAssert.assertTrue(webDriver.getTitle().contains("Home Page"));
 
         // 3. Perform login
         webDriver.findElement(By.id("user-icon")).click();
-        webDriver.findElement(By.id("name")).sendKeys("Roman");
-        webDriver.findElement(By.id("password")).sendKeys("Jdi1234");
+        webDriver.findElement(By.id("name")).sendKeys(TestProperties.getTestProperties().getProperty("login"));
+        webDriver.findElement(By.id("password")).sendKeys(TestProperties.getTestProperties().getProperty("password"));
         webDriver.findElement(By.id("login-button")).click();
 
         // 4. Assert Username is loggined
         softAssert.assertEquals(webDriver.findElement(By.id("user-name")).getText(), "ROMAN IOVLEV");
 
         // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
-        softAssert.assertEquals(webDriver.findElement(By.linkText("HOME")).getText(), "HOME");
-        softAssert.assertTrue(webDriver.findElement(By.linkText("HOME")).isDisplayed());
-
-        softAssert.assertEquals(webDriver.findElement(By.linkText("CONTACT FORM")).getText(), "CONTACT FORM");
-        softAssert.assertTrue(webDriver.findElement(By.linkText("CONTACT FORM")).isDisplayed());
-
-        softAssert.assertEquals(webDriver.findElement(By.linkText("SERVICE")).getText(), "SERVICE");
-        softAssert.assertTrue(webDriver.findElement(By.linkText("SERVICE")).isDisplayed());
-
-        softAssert.assertEquals(webDriver.findElement(By.linkText("METALS & COLORS")).getText(), "METALS & COLORS");
-        softAssert.assertTrue(webDriver.findElement(By.linkText("METALS & COLORS")).isDisplayed());
+        List<WebElement> actualHeaderNavBarItems =
+            webDriver.findElements(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']/li/a"));
```

Comment:
This locator is very vulnerable. If any class will be added test fails. Try to find better one.